src/main/kotlin/uk/gov/justice/digital/hmpps/pecs/jpc/pricing/importer/PricesSpreadsheet.kt

```diff
@@ -37,20 +37,22 @@ class PricesSpreadsheet(private val spreadsheet: Workbook,
     fun mapToPrice(row: Row) = getPrice(supplier, row)
 
     private fun getPrice(supplier: Supplier, row: Row): Price {
-        val journeyId = Result.runCatching { row.getCell(JOURNEY_ID).numericCellValue }.getOrElse { throw IllegalArgumentException("Error retrieving journey id for supplier '$supplier'", it) }
+        val journeyId = Result.runCatching { row.getCell(JOURNEY_ID).numericCellValue }.getOrElse { throw RuntimeException("Error retrieving journey id for supplier '$supplier'", it) }
         val fromLocationName = row.getFormattedStringCell(FROM_LOCATION)
-                ?: throw NullPointerException("From location name cannot be blank")
+                ?: throw RuntimeException("From location name cannot be blank")
         val toLocationName = row.getFormattedStringCell(TO_LOCATION)
-                ?: throw NullPointerException("To location name cannot be blank")
-        val price = Result.runCatching { row.getCell(PRICE).numericCellValue }.getOrElse { throw IllegalArgumentException("Error retrieving price for supplier '$supplier'", it) }
+                ?: throw RuntimeException("To location name cannot be blank")
+        val price = Result.runCatching { (row.getCell(PRICE).numericCellValue * 100).toInt() }
+                .onSuccess { if (it == 0) throw RuntimeException("Price must be greater than zero") }
```

Comment:
Including additional check to make sure price is greater than zero.  This is enforced on the JPA entity.  This makes it a little easier to consume if the error occurs.